include/bluetooth/mesh/sensor_types.h

```diff
@@ -44,6 +44,336 @@ extern "C" {
 		const struct bt_mesh_sensor_type *volatile __unused val =      \
 			&_type;                                                \
 	}
+
+/** @defgroup bt_mesh_sensor_formats_percentage Percentage sensor formats
+ *  @{
+ */
+
+/** Percentage 8
+ *  - Unit: Percent
+ *  - Encoding: 8 bit unsigned scalar (Resolution: 0.5)
```

Comment:
8/16/24/32 bits or 8-/16-/24-/32-bit
For all percentage sensor formats added in this file.

Reply:
I did this to match the docs for the sensor types below. Suggest a separate PR to fix this across the file?